Userland/Libraries/LibWasm/AbstractMachine/AbstractMachine.cpp

```diff
@@ -314,6 +329,13 @@ InstantiationResult AbstractMachine::instantiate(Module const& module, Vector<Ex
                         };
                         return;
                     }
+                    auto maybe_data_address = m_store.allocate_data(data.init);
+                    if (!maybe_data_address.has_value()) {
+                        instantiation_result = InstantiationError { "Failed to allocate a data instance for an active data segment" };
```

Comment:
Would these errors benefit from a sprinkling of `sv` literals?

Reply:
They're not in a perf-critical path, so I doubt it would be very useful, but might as well.

Reply:
done!